src/test/testSession.js

```diff
@@ -50,15 +50,23 @@ describe('Session', () => {
 
     it('marks accessed models', () => {
         const session = orm.session(emptyState);
-        expect(session.accessedModels).toHaveLength(0);
+        expect(session.accessedModels).toEqual({});
 
-        session.markAccessed(Book.modelName);
-        expect(session.accessedModels).toHaveLength(1);
-        expect(session.accessedModels[0]).toBe('Book');
+        session.markAccessed(Book.modelName, [0]);
 
-        session.markAccessed(Book.modelName);
+        expect(session.accessedModels).toEqual({
+            Book: {
+                0: true
+            }
+        });
 
-        expect(session.accessedModels[0]).toBe('Book');
+        session.markAccessed(Book.modelName, [1]);
+        expect(session.accessedModels).toEqual({
+            Book: {
+                0: true,
+                1: true
+            }
+        });
     });
 
```

Comment:
I guess we should also test if models are marked as accessed if they are resolved by the query function?